src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -390,11 +386,12 @@ export class ExposureNotificationService {
           lastCheckedPeriod,
         );
       }
+      return finalize({}, lastCheckedPeriod);
     } catch (error) {
       captureException('performExposureStatusUpdate', error);
     }
 
-    return finalize({}, lastCheckedPeriod);
+    return finalize();
```

Comment:
We only want to add lastCheckedPeriod ONLY if the framework returns summary. Although, what happens with Android framework version `18.0.2` (current) is that user will always force to download 14 days key every time until they get exposed. This behavior will be fixed automatically after consuming `18.0.3`.

Reply:
804 keys zip file is about 16 KB. That's the last 14 days keys on our server currently. So, it's not too bad to download every time when launch v1.